goss.yaml

```diff
@@ -0,0 +1,76 @@
+command:
+  airflow-version:
+    exec: "airflow version"
+    exit-status: 0
+    stdout:
+        - "/^3\\.1\\.[0-9]+\\s*$/"
```

Comment:
this regex is ok, but having `3.1.` would have been sufficient too. all elements of this stdout array are searched in the output